sklearn/neighbors/_kde.py

```diff
@@ -357,3 +408,41 @@ def sample(self, n_samples=1, random_state=None):
                 / np.sqrt(s_sq)
             )
             return data[i] + X * correction[:, np.newaxis]
+
+    def _more_tags(self):
+        return {
+            "_xfail_checks": {
+                "check_sample_weights_invariance": (
+                    "sample_weight must have positive values"
+                ),
+            }
+        }
```

Comment:
```suggestion
```
This should fix the error in the azure pipeline test.  You can also merge the main branch, because  `_more_tags` has been removed since.